charts/internal/shoot-system-components/charts/allow-egress/templates/allow-tcp-egress.yaml

```diff
@@ -0,0 +1,26 @@
+# this is a technical service created to mitigate an issue with the TCP egress traffic for Shoots using
+# Azure Standard LoadBalancers (see https://github.com/gardener/gardener-extension-provider-azure/issues/1)
+apiVersion: v1
+kind: Service
+metadata:
+  name: allow-tcp-egress
+  namespace: kube-system
+  annotations:
+    azure.remedy.gardener.cloud/ignore: "true"
+    gardener.cloud/description: |
+      This is a technical Service created to mitigate an issue with the TCP egress traffic for Shoots using
+      Azure Standard LoadBalancers (see https://github.com/gardener/gardener-extension-provider-azure/issues/1).
+      It is needed because the Standard LoadBalancers will block all outgoing TCP traffic if there is not at least
+      one open TCP Port. This is a temporary workaround and will be removed in a future version, once we have a
```

Comment:
Can we remove the statement for temporary workaround? I mean the NatGateway will stay optional, that means this service would be always required if there is no NatGateway used.